### ISSUE TITLE
Option to download Livestream 

### ISSUE THREAD

**Issue by Hanankhan1983** [TeamNewPipe/NewPipe#8595]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

No

### Why do you want this feature?

I want to download Livestream to get all episodes in one (all in one)

**Comment by mhmdanas** [comment]
Please fill out your feature description properly. It seems like ths issue is a duplicate of https://github.com/TeamNewPipe/NewPipe/issues/6232, but I can't be sure till you properly describe your feature request.

**Comment by Hanankhan1983** [comment]
> Please fill out your feature description properly. It seems like this issue is a duplicate of #6232, but I can't be sure till you properly describe your feature request.

Whenever I try to download Livestreams from YouTube it just download a 5 second or 3 second clip 
Pls add so we could download full Livestream 

I want this feature so I could download all episodes in one video
Any more questions 😃 pls feel free to ask

**Comment by Hanankhan1983** [comment]
> Please fill out your feature description properly. It seems like this issue is a duplicate of #6232, but I can't be sure till you properly describe your feature request.

Hello 👋  
I need this feature to download all episodes of a series in one video
I would really appreciate it if you could add this feature
Whenever I try to download Livestream it download a clip of 5 second or 3 second
Pls add this feature if you guys could:)
Feel free to ask anything

**Comment by opusforlife2** [comment]
What episodes?

**Comment by Hanankhan1983** [comment]
> What episodes?

Any Livestream episode

**Comment by opusforlife2** [comment]
> so I could download all episodes in one video

What you mean by "episode" in a live stream is not clear.

Do you mean simply downloading a livestream while it is still live?

**Comment by Hanankhan1983** [comment]
> > so I could download all episodes in one video
> 
> What you mean by "episode" in a live stream is not clear.
> 
> Do you mean simply downloading a livestream while it is still live?

Yes because there's Livestreams that are still live and we can't download it they just be live forever like cartoons replay continue

**Comment by opusforlife2** [comment]
Okay. I'm not sure how that will work, but your feature request is clear now.

**Comment by opusforlife2** [comment]
@Hanankhan1983 Edit the OP to fill the template properly.

**Comment by Hanankhan1983** [comment]
> @Hanankhan1983 Edit the OP to fill the template properly.
 
Okay I will fill it up correctly 🙂

**Comment by Hanankhan1983** [comment]
> @Hanankhan1983 Edit the OP to fill the template properly.

I rewrite it check ✅ it now
